runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -449,6 +448,10 @@ private LootTrackerBox buildBox(LootTrackerRecord record)
 				}
 			}
 		}
+		if (config.getIgnoredNpcs().contains(record.getTitle()))
```

Comment:
Shouldn't this be placed above the grouped loot block?

Reply:
We also probably shouldn't be doing a contains check on a string.